src/components/Notes/AddNote.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+
+const AddNote = ({ handleAddNote }) => {
+    const [expanded, setExpanded] = useState(false);
+    const [noteTitle, setNoteTitle] = useState('');
+    const [noteContent, setNoteContent] = useState('');
+
+    const handleExpand = () => {
+        setExpanded(true);
+    };
+
+    const handleCancel = () => {
+        setExpanded(false);
+        setNoteTitle('');
+        setNoteContent('');
+    };
+
+    const handleSaveClick = () => {
+        if (noteTitle.trim().length > 0 && noteContent.trim().length > 0) {
```

Comment:
Can we handle the form validation here, so let's show a validation message when the user missing field.